### PR TITLE
add helsinki hug 5 event blog

### PR DESCRIPTION
**Pull request by rubyrat**

### CODE REVIEW

File: src/posts/helsinki-hashicorp-user-group-meetup-5.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- ckurowski
+tags:
+- HashiCorp
+date: 2022-28-10
+image: "/blogs/helsinki-hug-5/hug-blog-event-5.png"
+featured: false
+
+---
+
+**Helsinki HashiCorp User Group (HUG) is a quarterly meetup full of interesting talks and insights about all things HashiCorp, plus a chance to connect with fellow professionals and hobbyists alike.**
```
Comment (Fleuri) [inline src/posts/helsinki-hashicorp-user-group-meetup-5.md:15]: It should be quarterly, but hasn't been. Should that word be dropped?

File: src/posts/helsinki-hashicorp-user-group-meetup-5.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+**Time:** Wednesday, 30 November, 17:00-22.00 EET
+
+**Place:** UpCloud, Aleksanterinkatu 15 B, Helsinki
+
+[Register here](https://www.meetup.com/helsinki-hashicorp-user-group/events/288940115/)
+
+
+The next Helsinki HashiCorp User Group is hosted by [UpCloud](https://upcloud.com/) at their offices in downtown Helsinki. This time round there’ll be a packed agenda with four presentations and some very special guests. Join for an evening of talks, networking, and snacks! The talks will also be live streamed and recorded for those who prefer to join online.
```
Comment (Fleuri) [inline src/posts/helsinki-hashicorp-user-group-meetup-5.md:24]: Amir's talk isn't really a presentation, but more of welcoming words & greetings.

**Review by Fleuri — CHANGES_REQUESTED** [review]
Some comments added. Please review and PR again.

File: src/posts/helsinki-hashicorp-user-group-meetup-5.md (hunk 1 above)
Comment (rubyrat) [reply src/posts/helsinki-hashicorp-user-group-meetup-5.md:15]: yeah, I guess it's only twice this year. Will amend that! Still pretty active though :)

File: src/posts/helsinki-hashicorp-user-group-meetup-5.md (hunk 2 above)
Comment (rubyrat) [reply src/posts/helsinki-hashicorp-user-group-meetup-5.md:24]: I can rephrase, so it's after the speakers - 'Plus some welcoming words from...'

**Pull request merged**